guard-ffi/src/errors.rs

```diff
@@ -29,6 +29,7 @@ fn get_code(e: &Error) -> ErrorCode {
         Error::MissingValue(_err) => 16,
         Error::FileNotFoundError(_) => 17,
         Error::IllegalArguments(_) => 18,
+        Error::InternalError(_) => unreachable!(),
```

Comment:
Do we not need to assign a code to this error?

Reply:
No, since this error is supposed to be internal only, it should be unreachable